src/trc.py

```diff
@@ -259,7 +286,7 @@ def save(self, filename):
 
             for frame in self['Frame#']:
                 time, line_data = self[frame]
-                values = [f'{v:.5f}' for values in line_data for v in values]
+                values = ['' if math.isnan(v) else f'{v:.5f}' for line_data in line_data for v in line_data]
```

Comment:
I don't think having `for line_data in line_data` is very good.

Reply:
Oops! Thanks for catching that.